Cargo.toml

```diff
@@ -17,10 +17,14 @@ clap = { version = "4.0.29", features = ["derive"] }
 fast_paths = "0.2.0"
 rust_road_router = { git = "https://github.com/kit-algo/rust_road_router" }
 mpi = "0.6"
+geo = "0.23.1"
 # dependencies for serialization with protobuf
 prost = "0.11.5"
 # bytes replaces the signature of std::io::Cursor to implemnt Buf somehow...
 bytes = "1.3.0"
+# makes sure that marked tests are run sequentially (needed for tests that use the InertialFlowCutter because
+# they use the same temporary output directory)
+serial_test = "0.10.0"
```

Comment:
Interesting. What happens if you use `IntertialFlowCutter` in mutliple threads/processes from within the application?

Reply:
Have you tried running the `metis-test` tests in serial? maybe the results will be more predictable here as well 🤔

Reply:
The problem was, that the flow cutter adapter uses the same temporary output directory for the tests. And especially when one test cleans these directories and other use them yet, that's a problem. But when I think about it I just could change the output directories

Reply:
In the main matsim project we use the convention `./test_output/our/package/YourModule/test_method_name` as folder to put files in. I have sticked to that in my unit tests so far. Maybe this is something your could do with the tmp files as well.